catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -207,29 +207,29 @@
             Metacard.METADATA,
             true /* indexed */,
             true /* stored */,
-            false /* tokenized */,
+            true /* tokenized */,
             false /* multivalued */,
             XML_TYPE));
     descriptors.add(
         new AttributeDescriptorImpl(
             Metacard.RESOURCE_URI,
             true /* indexed */,
             true /* stored */,
-            false /* tokenized */,
+            true /* tokenized */,
             false /* multivalued */,
             STRING_TYPE));
     descriptors.add(
         new AttributeDescriptorImpl(
             Metacard.RESOURCE_DOWNLOAD_URL,
```

Comment:
❓ I noticed that we have changed indexed, stored, and tokenized on several attributes.  Are we concerned about this impacting ingest or query performance at all?

Reply:
I did some rough tests of ingest time and it didn't seem to affect it. The changes made here where just to make the attributes consistent within our api.